search/views_test.py

```diff
@@ -212,7 +212,9 @@ def test_query_filters(self):
                     "fields": [
                         "profile.first_name.folded",
                         "profile.last_name.folded",
-                        "profile.preferred_name.folded"
+                        "profile.preferred_name.folded",
+                        "profile.username",
```

Comment:
I am getting weird behavior when doing a prefix search on a username which includes dots. Can we create a folded version and use that to index?